src/System.Private.Xml.Linq/src/System.Private.Xml.Linq.csproj

```diff
@@ -19,6 +19,9 @@
     <Compile Include="$(CommonPath)\CoreLib\System\Text\StringBuilderCache.cs">
       <Link>System\Text\StringBuilderCache.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Marvin.cs">
+      <Link>System\Marvin.cs</Link>
```

Comment:
We don't need to pull in our own copy of `Marvin`. All APIs we need are already publicly accessible via the `string` or `HashCode` types.